src/structures/MessageEmbed.js

```diff
@@ -327,6 +346,21 @@ class MessageEmbed {
       } : null,
     };
   }
+
+  /**
+   * Checks for valid field input and resolves strings
+   * @param {StringResolvable} name The name of the field
+   * @param {StringResolvable} value The value of the field
+   * @param {boolean} [inline=false] Set the field to display inline
+   * @returns {EmbedField}
+   */
+  static checkField(name, value, inline = false) {
+    name = Util.resolveString(name);
+    if (!String(name)) throw new RangeError('EMBED_FIELD_NAME');
+    value = Util.resolveString(value);
+    if (!String(value)) throw new RangeError('EMBED_FIELD_VALUE');
```

Comment:
What does `!String(value)` achieve here?  `Util.resolveString` already converts the output into a string, so it seems like just checking for falsy should be enough.

Reply:
Don't know, just copied and pasted from addField to this method. I will look into seeing if that's just meant to be a falsy check.